.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.4
```

Comment:
Do we want a specified ruby-version if this is a gem?

Reply:
that crossed my mind as well, but I am hesitant to do that in this PR. I'll open an issue.